package.json

```diff
@@ -346,7 +346,9 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "6.27.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
+    "@solana/web3.js": "1.36.0",
```

Comment:
jfyi looks like this is out of date: https://solana-labs.github.io/solana-web3.js/

Reply:
probably fixed by rebase on https://github.com/brave/brave-core/pull/13949/files

Reply:
solana just fixed another potential sec issue, so if possible we should update it to a version that has https://github.com/solana-labs/solana/pull/26282 as part of this PR